src/app/components/ElectronicDelivery/ElectronicDelivery.jsx

```diff
@@ -168,73 +169,75 @@ class ElectronicDelivery extends React.Component {
     const searchKeywords = this.props.searchKeywords;
 
     return (
-      <div id="mainContent">
-        <div className="nypl-request-page-header">
-          <div className="row">
-            <div className="content-wrapper">
-              <Breadcrumbs
-                query={searchKeywords}
-                type="edd"
-                bibUrl={`/bib/${bibId}`}
-                itemUrl={`/hold/request/${bibId}-${itemId}`}
-              />
-              <h2>{appConfig.displayTitle}</h2>
+      <DocumentTitle title={`${title} | Shared Collection Catalog | NYPL`}>
```

Comment:
Okay, Courtney updated #675 with EDD information: `Electronic Delivery Request | Shared Collection Catalog | NYPL`. This should be the last update!